dockerfiles/Dockerfile.source

```diff
@@ -0,0 +1,22 @@
+# --------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------
+# Dockerfile to run ONNXRuntime with source build for CPU
+
+# Ubuntu 16.04 Base Image
+FROM ubuntu:16.04
+WORKDIR /code
+ENV PATH /opt/miniconda/bin:/code/cmake-3.14.3-Linux-x86_64/bin:${PATH}
+
+# Dependencies: Ubuntu core, conda, cmake
+COPY install_common_deps.sh /code/install_common_deps.sh
```

Comment:
it would be better to call the script that's kept in the repo. otherwise, where does the script is frozen and only exists in the image?

Reply:
Good point! Fixed.